sky/shell/platform/mojo/platform_view_mojo.cc

```diff
@@ -147,5 +156,23 @@ void PlatformViewMojo::OnEvent(mojo::EventPtr event,
   callback.Run();
 }
 
+void PlatformViewMojo::Create(
+    mojo::ApplicationConnection* connection,
+    mojo::InterfaceRequest<::keyboard::KeyboardService> request) {
+
+  mojo::ServiceProviderPtr sp;
+  connector_->ConnectToApplication("mojo:keyboard", mojo::GetProxy(&sp), nullptr);
+
+#ifdef OS_LINUX
```

Comment:
I'd flip this around so that it's OS_ANDROID that's special cased.  It's Android that's crazy here.